diff --git a/app/boards/shields/nice_view/widgets/peripheral_status.c b/app/boards/shields/nice_view/widgets/peripheral_status.c
index e9002b33..c75aa6cf 100644
--- a/app/boards/shields/nice_view/widgets/peripheral_status.c
+++ b/app/boards/shields/nice_view/widgets/peripheral_status.c
@@ -116,7 +116,7 @@ int zmk_widget_status_init(struct zmk_widget_status *widget, lv_obj_t *parent) {
 
     lv_obj_t *art = lv_img_create(widget->obj);
     bool random = sys_rand32_get() & 1;
-    lv_img_set_src(art, random ? &balloon : &mountain);
+    lv_img_set_src(art, &balloon);
     lv_obj_align(art, LV_ALIGN_TOP_LEFT, 0, 0);
 
     sys_slist_append(&widgets, &widget->node);
